setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = skt
+description = "Sonic Kernel Testing"
+long_description = file: README.md
+version = 0.0.1
+author = Red Hat, Inc.
+license = GPL
```

Comment:
Should this be GPLv2+?

Reply:
Yes, GPLv2+ is the right license. Quoting one of the headers `GNU General Public License v.2 or later.`

Reply:
Ah, good catch. I'll fix that up.